src/bin/lpython.cpp

```diff
@@ -1129,6 +1129,7 @@ int main(int argc, char *argv[])
         app.add_flag("--static", static_link, "Create a static executable");
         app.add_flag("--no-warnings", compiler_options.no_warnings, "Turn off all warnings");
         app.add_flag("--no-error-banner", compiler_options.no_error_banner, "Turn off error banner");
+        app.add_flag("--enable-bounds-checking", compiler_options.enable_bounds_checking, "Turn on index bounds checking");
```

Comment:
I think its default value should be `true` according to LPython contract defined in https://github.com/lcompilers/lpython/issues/1139 (see quotation below). However if I enable `--fast` option then I think this should be `false` by default. Note that I think at the end user's choices should be valued the most. If they enable `--fast --enable-bounds-checking` then `compiler_options.enable_bounds_checking` should be `true`. See the quotation below,

> The default option in LPython should be Debug mode. The Debug mode must have all runtime safety options on. In this Debug mode, you get very fast compilation, and safe runtime behavior. No silent surprises. Beautiful runtime error messages with stack traces. If you don't get any runtime error and things compiles (obviously), then it will behave just like in CPython.